### DISCUSSION TITLE
Is there a way to automatically import bank account balance changes using custom API?

### DISCUSSION THREAD

**Discussion by tukusejssirs** [firefly-iii/firefly-iii#9081] [Firefly III]
I’d like to create some Node.js apps to get the data directly from banks I use. That should be possible, however, is Firefly III capable of polling the data every n minutes or do I need to schedule it externally import it into Firefly III?

Or is there a plugin system in Firefly III that I could leverage?

Thanks! :pray:

**Comment by JC5** [comment]
Did you read the documentation?
